_cucumber/features/support/env.rb

```diff
@@ -0,0 +1,60 @@
+$: << File.dirname(__FILE__)+'/../../lib'
+
+require 'capybara'
+require 'capybara/cucumber'
+require 'capybara/poltergeist'
+require 'rspec'
+require 'capybara/poltergeist'
+require 'selenium-webdriver'
+require 'require_all'
+require 'fileutils'
+require 'rubocop'
+require 'capybara-screenshot/cucumber'
+
+require_relative 'app'
+Dir["#{File.dirname(__FILE__)}/../../lib/pages/*.rb"].each { |page| load page }
```

Comment:
This line is causing issues on certain systems. The ordering of the imports is not guarenteed. base.rb must be loaded first